### ISSUE TITLE
data inconsistency

### ISSUE THREAD

**Issue by tessus** [influxdata/influxdb-relay#69]
I was looking at the architecture and I'm a bit puzzled by the fact that data inconsistency can occur and I'm wondering how I'm supposed to fix those.

Here's an example:

- client writes to relay
- relay writes to InfluxDB A and InfluxDB B
- InfluxDB A write is successful
- InfluxDB B write is not successful
- client got success message, because the write to A was first

So now I have 2 databases with different data. Although A has all the correct and complete data. 
Now image the same process as above again, but B gets the successful write. In this case now both databases have incorrect and incomplete data.

Or am I missing something here?

**Comment by kashifmin** [comment]
@tessus You're right. I've noticed the same. Do you know of any alternatives?

**Comment by tessus** [comment]
Nope, but I guess that's the reason why they are selling the influxdb-cluster. They don't even want to solve this in the free relay code....

**Comment by rockyluke** [comment]
Please take a look on https://github.com/veepee-moc/influxdb-relay

**Comment by jhg03a** [comment]
That's a lot of really interesting work @rockyluke, was there ever a PR to upstream it?
